docs/schemas/Library-Minimal-Metadata/Library Minimal Metadata.csv

```diff
@@ -0,0 +1,52 @@
+library pool fmol,af1a6f3f-aca9-4452-b86e-f3c70c3600b6,,,Amount of library pool in the tube as measured in femtamoles (fmol),,,FALSE,,Integer,1,9/9/23,Delivery Library Pool,Library Minimal Metadata,,,,
+Loading concentration pM,a1a1b046-549d-4e94-9b3c-5fac2a31fdd6,,,Sequencer Loading Concentration as measured in pM (pmol/L).  This is a value used by the SeqCore.,,,FALSE,,Float,1,9/9/23,Delivery Library Pool,Library Minimal Metadata,,,,
+Length of Read 2 (for Paired End Runs) ,0b9a9cbc-b8dd-42a2-a567-aafd9370db30,,,"Separate field to replace the combined field ""Sequencing cycle"". Sequencing Cycle is needed for sequencing the library pool.  The sequencing cycle needed is specific to the Library Chemistry Method and is required instruction to the SeqCores.",,,TRUE,,Integer,1,9/9/23,Delivery Library Pool,Library Minimal Metadata,,,,
+Length of Index 1 (i7 Primer) ,0b94df0d-d96e-4d49-a498-b3eb83afc5f8,,,"Separate field to replace the combined field ""Sequencing cycle"". Sequencing Cycle is needed for sequencing the library pool.  The sequencing cycle needed is specific to the Library Chemistry Method and is required instruction to the SeqCores.",,,TRUE,,Integer,1,9/9/23,Delivery Library Pool,Library Minimal Metadata,,,,
+Length of Index 2 (i5 Primer) ,a61c1d55-b880-499c-ba4a-30311fdca62f,,,"Separate field to replace the combined field ""Sequencing cycle"". Sequencing Cycle is needed for sequencing the library pool.  The sequencing cycle needed is specific to the Library Chemistry Method and is required instruction to the SeqCores.",,,TRUE,,Integer,1,9/9/23,Delivery Library Pool,Library Minimal Metadata,,,,
+Length of Read 1,f22ac08a-bb81-4524-91f0-0e1e1a032335,,,"Separate field to replace the combined field ""Sequencing cycle"". Sequencing Cycleis  needed for sequencing the library pool.  The sequencing cycle needed is specific to the Library Chemistry Method and is required instruction to the SeqCores.",,,TRUE,,Integer,1,9/9/23,Delivery Library Pool,Library Minimal Metadata,,,,
+library pool tube contents nM,32f2d02b-7300-4554-aa93-6de6e456eda7,library pool concentration (nM),,Library pool concentration as measured in nanomolarity (nMol/L),,,FALSE,,Float,1,9/9/23,Delivery Library Pool,Library Minimal Metadata,,,,
+Library pool tube volume ul,b1b923ac-c218-4db4-a3b1-45a219612567,,,Library pool volume as measured in ul,,,FALSE,,Integer,1,9/9/23,Delivery Library Pool,Library Minimal Metadata,,,,
+PhiX spike in percent,b5ab26ad-d523-406e-a85b-e77f2f4b06b5,,,"PhiX spike-in percent desired to be added to the library pool for sequencing.  PhiX is used to increase complexity of the sample being sequenced, to reduce sequencing artifacts maintain sequencing quality on the instruments.  This is an optional instruction to the SeqCore.",,,FALSE,,Integer,1,9/9/23,Delivery Library Pool,Library Minimal Metadata,,,,

```

Comment:
Usually percentages should be a float, but I don't know what PhiX spike is and if it is guaranteed to be an integer.